docs/articles_en/learn-openvino/large-language-models/ov-api-vs-optimum.rst

```diff
@@ -0,0 +1,132 @@
+.. {#native_vs_hugging_face_api}
+
+Hugging Face vs Native OpenVINO API
+========================================
+
+.. meta::
+   :description: Explore learning materials, including interactive
+                 Python tutorials and sample console applications that explain
+                 how to use OpenVINO features.
+
+
+.. toctree::
+   :maxdepth: 1
+   :hidden:
+
+   Loading and Optimizing LLMs with Optimum Intel <gen_ai_guide>
+   LLM Inference with Optimum Intel <llm_inference>
+   LLM Inference with OpenVINO API <llm_inference_native_API>
+   LLM Inference with Native OpenVINO <llm_inference_native_ov>
+
+
+Large Language Models (LLMs) like GPT are transformative deep learning networks
+capable of a broad range of natural language tasks, from text generation to language translation.
+OpenVINO optimizes the deployment of these models, enhancing their performance and integration into various applications.
+This guide shows how to use LLMs with OpenVINO, from model loading and conversion to advanced use cases.
+
+The advantages of using OpenVINO for LLM deployment:
+
+●	OpenVINO requires fewer dependencies than frameworks like
```

Comment:
Can we unify the style of all the items in the list, I mean put the main message in bold first and provide details, then.

Reply:
changed